release/rancher/rancher.go

```diff
@@ -547,7 +555,7 @@ func dockerImagesDigests(imagesFileURL, registry string) (imageDigest, error) {
 		if imageAndVersion == "" || imageAndVersion == " " {
 			continue
 		}
-		slog.Info("image: " + imageAndVersion)
+		log.Println("image: " + imageAndVersion)
```

Comment:
Library code shouldn't be writing anything. The sapling code of the library should have the freedom to decide what to do with returned error output and calling code or app code should indicate output format.